### ISSUE TITLE
"cargo run bin" runs my program three times at once

### ISSUE THREAD

**Issue by radix** [AtomBuild/atom-build-cargo#73]
I'm not sure if this is a bug in atom-build-cargo or atom-build, but when I use the "Cargo Run Bin" Atom command, three copies of my program are spawned at once, with all of their output going to the same build output pane.

I'm using Windows 10.

**Comment by radix** [comment]
I think I figured out that this was happening because I happened to have two extra folders added to my project tree, all of which contained cargo projects. They were for completely different apps, though, and only the first one was actually executed (albeit three times).

**Comment by alygin** [comment]
It looks very similar to [this issue](https://github.com/noseglid/atom-build/issues/460) in Atom Build. Does it happen always? Does it happen only for "Cargo run bin"?

**Comment by radix** [comment]
Yes, I can reproduce it easily by adding an additional folder to my side-bar project tree. It has to be a folder containing a Cargo.toml - any other folder won't cause the bug. And, if I remove the additional folders, it continues to run extra copies of my program until I restart atom.
